riva/proto/riva_nmt.proto

```diff
@@ -82,6 +82,9 @@ message SynthesizeSpeechConfig {
   int32 sample_rate_hz = 2;
   string voice_name = 3;
   string language_code = 4;
+  string rate = 5;
+  string pitch = 6;
+  string volume = 7;
```

Comment:
putting a comment here regarding field name as discussing on other PR. Whether to call it `prosody_rate` to be clearly distinguishable from `sample_rate_hz`